src/coreclr/src/jit/gentree.h

```diff
@@ -2959,6 +2959,23 @@ struct GenTreeIntCon : public GenTreeIntConCommon
     // sequence of fields.
     FieldSeqNode* gtFieldSeq;
 
+#ifdef DEBUG
+    // If the value represents target address, holds the method handle to that target which is used
+    // to fetch target method name and display in the disassembled code.
+    CORINFO_METHOD_HANDLE gtMethodHandle = (CORINFO_METHOD_HANDLE)Unknown;
+
+    // Enum values are such that they don't overlap with helper's or user function's method handle.
+    enum MethodHandleType{Unknown                  = 2,
```

Comment:
The `gtFlags` already contains `GTF_ICON_*_HDL` (everything under `GTF_ICON_HDL_MASK`). So is that sufficient, without needing this enum? Then, this `gtMethodHandle` could potentially be a union of different handle types that is interpreted based on that `GTF_*` value?

Reply:
Yes, didn't realize that I can just use `gtFlags`.